src/android/Text.tsx

```diff
@@ -48,10 +49,6 @@ export class Text extends CommonText {
             </RN.Text>
         );
     }
-
```

Comment:
Did you intend to delete this, or was it a merge error? Amay recently added this code.